### PR TITLE
MP-4623 Added backport for MerchantFacade::get method

### PR DESCRIPTION
**Pull request by volodymyr-bezuhlyi**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: src/Spryker/Zed/Merchant/Persistence/MerchantRepository.php [hunk 1]
```diff
@@ -78,4 +137,108 @@ public function hasKey(string $key): bool
...
             ->exists();
     }
+
+
+    /**
+     * @param int[] $merchantIds
+     *
+     * @return \Generated\Shared\Transfer\UrlTransfer[][]
+     */
+    public function getUrlsMapByMerchantIds(array $merchantIds): array
```
Comment (zssamoylov) [inline src/Spryker/Zed/Merchant/Persistence/MerchantRepository.php]:
```suggestion
    public function getGroupedUrlsByMerchantIds(array $merchantIds): array
```
what do you think?

File: src/Spryker/Zed/Merchant/Persistence/MerchantRepository.php [hunk 2]
```diff
@@ -78,4 +137,108 @@ public function hasKey(string $key): bool
...
+
+    /**
+     * @param \Propel\Runtime\ActiveQuery\ModelCriteria $query
+     * @param \Generated\Shared\Transfer\PaginationTransfer|null $paginationTransfer
+     *
+     * @return mixed|\Propel\Runtime\ActiveRecord\ActiveRecordInterface[]|\Propel\Runtime\Collection\Collection|\Propel\Runtime\Collection\ObjectCollection
+     */
+    protected function getPaginatedCollection(ModelCriteria $query, ?PaginationTransfer $paginationTransfer = null)
+    {
+        if ($paginationTransfer !== null) {
```
Comment (zssamoylov) [inline src/Spryker/Zed/Merchant/Persistence/MerchantRepository.php]:
```suggestion
        if (!$paginationTransfer) { return...
```
please use early return here.

File: src/Spryker/Zed/Merchant/Persistence/Propel/Mapper/MerchantMapper.php [hunk 3]
```diff
@@ -44,4 +50,54 @@ public function mapEntityToMerchantTransfer(
...
+
+
+    /**
+     * @param \Orm\Zed\Merchant\Persistence\SpyMerchant[] $merchantEntities
+     * @param \Generated\Shared\Transfer\MerchantCollectionTransfer $merchantCollectionTransfer
+     *
+     * @return \Generated\Shared\Transfer\MerchantCollectionTransfer
+     */
+    public function mapMerchantCollectionToMerchantCollectionTransfer(
+        $merchantEntities,
```
Comment (zssamoylov) [inline src/Spryker/Zed/Merchant/Persistence/Propel/Mapper/MerchantMapper.php]: please add type for `$merchantEntities`
```suggestion
        array $merchantEntities,
```

**Review by zssamoylov — COMMENTED** [review]
Looks like the branch names are wrong, please check with this doc 
https://github.com/spryker/spryker/wiki/Pull-Requests#hotfixing-via-nonsplit

**Comment by zssamoylov** [comment]
Please also create Release group and add changelogs for this PR.

**Comment by zssamoylov** [comment]
Please add release table using this url https://release.spryker.com/release/hotfix
And rename the branch, minor = ` hotfix/mp-4623-merchant-1-2-2`, patch = ` hotfix/mp-4623-merchant-1-1-3`

**Review by zssamoylov — APPROVED** [review]

**Comment by dereuromark** [comment]
The branches should never contain hotfix/ word
But they can stay for now if you use template, as it overwrite branch data.

**Pull request closed without merging**